code/ZAS/Airflow.dm

```diff
@@ -103,7 +103,7 @@ atom/movable/GotoAirflowDest(n)
 	return TRUE
 
 /mob/living/carbon/human/check_airflow_movable(n)
-	if(reagents.has_reagent(MEDCORES))
+	if(reagents.has_reagent(MEDCORES, SOFTCORES))
```

Comment:
this won't work, has_reagent doesn't accept a second id as an arg, the second arg in has_reagent is a minimum amount

just do it the way I said the comment before, which is also more maintainable

Reply:
booooo i thought i got it in fewer lines and you were gonna be like wow good job robin you are such an innovative thinker and then give me a gold star

anyways THAT should do it

Reply:
OH DEAR FUCKING GOD

Reply:
okay no its good for real this time ahahahaha..